compliance_checker/cf/cf_1_7.py

```diff
@@ -501,6 +504,7 @@ def _check_gmattr_existence_condition_ell_pmerid_hdatum(self, var):
                     "reference_ellipsoid_name",
                     "prime_meridian_name",
                     "horizontal_datum_name",
+                    "geographic_crs_name",
```

Comment:
Add test coverage for added functionality.

Reply:
in progress ...